e2e/run-suite

```diff
@@ -14,20 +14,28 @@ CMD="start"
 PARAMS=""
 SLOWMO=0
 URL=${BASE_URL:-"http://$HOST:$PORT"}
+integrationFolder=../../e2e 
 
-if [ "$1" == "debug" ]; then
-    echo -e "Debug mode"
-    SLOWMO=1
-    PARAMS="--no-exit"
-fi
-
-if [ "$1" == "dev" ]; then
-    echo "Dev mode"
-    CMD="open"
-fi
 
 cd packages/grafana-e2e
 
+case "$1" in 
+    "debug")
+        echo -e "Debug mode"
+        SLOWMO=1
+        PARAMS="--no-exit"
+        ;;
+    "dev")
+        echo "Dev mode"
+        CMD="open"
+        ;;
+    "")
+        ;;
+    *)
+        integrationFolder=../../e2e/"${args[0]}"
+        ;;
+esac
```

Comment:
this is a maybe not ideal, but suite name "dev" means open cypress with all suite folders, and when no suite name specified we will run all specs